designer/client/src/actions/actionTypes.ts

```diff
@@ -10,8 +10,8 @@ export type ActionTypes =
     | "DELETE_NODES"
     | "NODES_CONNECTED"
     | "NODES_DISCONNECTED"
-    | "STICKY_NOTES_UPDATED"
-    | "STICKY_NOTE_DELETED"
+    | "STICKY_NOTE_UPDATED"
```

Comment:
these actions have payload and should be typed https://github.com/TouK/nussknacker/blob/6f763e62e80749cf20e0bc1bd9b2e17b70c55f6d/designer/client/src/actions/reduxTypes.ts#L17